src/components/mortgage-investment-compare/index.js

```diff
@@ -23,17 +23,21 @@ import {
 const COMPOUND_FREQUENCY = 12
 
 class MortgageInvestmentCompare extends React.Component {
-  constructor () {
-    super()
+  constructor (props) {
+    super(props)
```

Comment:
What props are we giving this component?

Reply:
it's just a good practice and something I do every time I create a class. I've run into stupid bugs in the past for not doing so: https://overreacted.io/why-do-we-write-super-props/.